.github/workflows/spark-integration-test.yml

```diff
@@ -57,6 +57,7 @@ jobs:
       matrix:
         architecture: [linux/amd64]
         java-version: [ 8, 11, 17 ]
+        scala-version: [ 2.12, 2.13 ]
```

Comment:
will change to 2.12 when every thing is ready to merge

Reply:
I think 2.12 only should be enough, otherwise, we will have too many CI pipeline.

Reply:
done